src/LocalComponents/actorStateStore.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+spec:
+  type: state.azure.cosmosdb
+  version: v1
+  metadata:
+  - name: url
+    value: https://localhost:8081/
+  - name: masterKey
+    value: C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==
```

Comment:
can/should we store this on user secrets?

Reply:
this is not really secret... MS publicly documents this static value (that's just how the emulator works)

e.g.

https://learn.microsoft.com/en-us/azure/cosmos-db/emulator